packages/apollo-cli/src/commands/queries/extract.ts

```diff
@@ -0,0 +1,80 @@
+import 'apollo-codegen-core/lib/polyfills';
+import { Command, flags } from '@oclif/command';
+import * as Listr from 'listr';
+import { print } from 'graphql';
+import { sha512 } from 'js-sha512';
```

Comment:
Was there a particular reason for using [`js-sha512`](https://nvm.im/js-sha512), a JavaScript implementation of the SHA-512 routine, rather than Node.js’ native [`crypto.createHash`](
https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options) (which is implemented in C/C++).